src/fragments/start/getting-started/angular/setup.mdx

```diff
@@ -115,8 +123,8 @@ import { AppComponent } from './app.component';
   imports: [
     BrowserModule,
     AppRoutingModule,
-    /* configure app with AmplifyUIAngularModule */
-    AmplifyUIAngularModule
+    /* configure App with AmplifyUIAngularModule */
```

Comment:
Should this comment still be here or reference the new `AmplifyAuthenticatorModule?

Reply:
You're right!
```suggestion
    /* configure App with AmplifyAuthenticatorModule */
```